composer.json

```diff
@@ -45,8 +45,13 @@
         "aist/aist-git-tools": "Show you informations about current GIT repository"
     },
     "autoload": {
-        "psr-0": {
-            "ZendDeveloperTools": "src/"
+        "psr-4": {
+            "ZendDeveloperTools\\": "src/"
+        }
+    },
+    "autoload-dev": {
```

Comment:
now the section "autoload-dev" is duplicated, please remove the section below ....